entrypoint.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+dos2unix entrypoint.sh
```

Comment:
does this affect people already on unix systems? Also, is this something you had to install on your computer?